### PR TITLE
EOS-13393: fix nodes fencing issue on sporadic hax stop timeouts

### PR DESCRIPTION
**Pull request by andriytk**
Currently, /var/motr is mounted before hax service startup and
unmounted after hax service stop. This mounting/unmounting is done
in hax systemd service unit file. So when hax fails to stop within
30 secs timeout (configured in Pacemaker), the node will be fenced
by Pacemaker to avoid the situation when /var/motr is mounted on
both nodes at the same time (which would lead to data corruption).

Hax stopping can be delayed for different reasons, mainly due to
bugs in Motr which are not easy to fix quickly (for example, hax
calls rconfc_start in fs-stats thread and it may get stuck if confd
process dies and does not come up during the node shutdown). So
we are solving it from another side:

1) Create separate resource in Pacemaker for /var/motr mgmt and
   stop doing it from hax systemd unit file.
2) Add this resource into the Motr Pacemaker group just before hax
   so that /var/motr mount is ready before hax startup (and unmounts
   after hax is stopped).
3) Set 5 seconds systemd stop timeout for hax (in similar way to
   all the rest Motr systemd services stop timeout) so that hax is
   killed by systemd if it cannot stop within this timeframe (for
   whatever reason) and it's not affecting the cluster state.

Tested manually:
 - build-ha-io PASS;
 - failback/failover PASS;
 - build-ees-ha-update PASS (with a minor fix related to #123).

### CODE REVIEW

**Comment by andriytk** [comment]
@mssawant , @chumakd , @azheregelya - this is the new PR instead of #140.

**Review by azheregelya — APPROVED** [review]
Looks good.

File: conf/script/build-ha-io [hunk 1]
```diff
@@ -517,6 +524,16 @@ motr_kernel_rsc_add() {
                                            kind=Optional symmetrical=false
 }
 
+var_motr_rsc_add() {
+    log "${FUNCNAME[0]}: Adding var-motr mounts to Pacemaker..."
+    sudo pcs -f $cib_file resource create var-motr1 ocf:heartbeat:Filesystem \
```
Comment (mssawant) [inline conf/script/build-ha-io:529]: need to add this resource to cleanup as well, prov-ha-reset script.

Comment (andriytk) [reply conf/script/build-ha-io:529]: I think it will be cleaned as part of c1/2 group there (the same was as hax and others).

Comment (mssawant) [reply conf/script/build-ha-io:529]: ah  right!

**Review by mssawant — APPROVED** [review]

**Comment by andriytk** [comment]
@ajaysrivas it's ready to be landed.

**Review by chumakd — APPROVED** [review]

**Review by indrajitzagade — APPROVED** [review]
Approving based on other approvals from hare team.

**Comment by indrajitzagade** [comment]
@andriytk Please fill the PR template going forward.

**Comment by indrajitzagade** [comment]
Please rebase since we cannot merge this yet.

**Pull request merged**